src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/ReadDescriptorsValidation.feature

```diff
@@ -124,7 +124,7 @@ Feature: Read a Descriptor
         # DMS-89
         @ignore
```

Comment:
Does this still have t be ignored?

Reply:
Yeah, was just changing to equal sign like the non-ignored one